examples/chart/teleport-cluster/values.yaml

```diff
@@ -658,10 +658,19 @@ postStart:
 
 # Resources to request for the teleport container
 # https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
+#
+# DANGER: Setting CPU limits is an antipattern and harmful in most cases.
```

Comment:
Can we also add this danger warning to NOTES.txt?

Reply:
I'll sent a second PR with a new value `dangerZoneIKnowWhatImDoing` with a flag to disable the CPU warning, and will move `teleportVersionOverride` there (because we keep having people controlling the Teleport version with this flag, even within Teleport).